apps/api/src/services/form.service.ts

```diff
@@ -23,6 +23,72 @@ export class FormService {
 		return await FormModel.getAllFormsWithSubmissionsCount(eventId);
 	}
 
+	async deleteFormField(formId: string, fieldId: string) {
+		try {
+			const formFieldInfo = await FormFieldsModel.findOne({
+				id: fieldId,
+				fk_form_id: formId,
+			});
+			if (!formFieldInfo) {
+				throw FFError.notFound(
+					`${SystemTable.FormFields}: Query Failed : 
+          form with id ${formId} could not be found`,
+				);
+			}
+
+			await FormFieldsModel.delete({
+				id: fieldId,
+			});
+
+			return {
+				ok: true,
+				message: 'Field deleted successfully',
+			};
+		} catch {
+			throw new InternalServerErrorException();
+		}
+	}
+
+	async editFormField(data: EditFormFieldDto, formId: string, fieldId: string) {
+		try {
+			const formInfo = await FormModel.findOne({
+				id: formId,
+			});
+			if (!formInfo) {
+				throw FFError.notFound(
+					`${SystemTable.FormFields}: Query Failed : 
+          form with id ${formId} could not be found`,
+				);
+			}
```

Comment:
Not sure, why we are doing it here. Let's move this to an middleware to check user has permission and if the resource exists